src/main/java/com/conveyal/r5/profile/Propagater.java

```diff
@@ -70,7 +78,9 @@ public Propagater (int[][] travelTimesToStopsEachIteration, int[] nonTransferTra
             for (int stop = 0; stop < travelTimesToStopsThisIteration.length; stop++) {
                 int travelTimeToStop = travelTimesToStopsThisIteration[stop];
 
-                if (travelTimeToStop == RaptorWorker.UNREACHED) continue;
+                // don't bother to even propagate stops that take too long to get to
+                // this will also skip unreached stops
+                if (travelTimeToStop > cutoffSeconds) continue;
```

Comment:
If I understand correctly, this means that the Propagater can no longer be used to get all the travel times, and like the PerTargetPropagater is no longer in any way generic: it only makes sense for reducer functions with very specific properties (ones that have no effect when all travel times are above the cutoff). If it is not generic, why does it accept reducer function references instead of just defining them as named methods on the Propagator class?